taiga_halo2/Cargo.toml

```diff
@@ -15,7 +15,7 @@ halo2_proofs = {version="0.3", features = ["dev-graph"]}
 bitvec = "1"
 subtle = { version = "2.3", default-features = false }
 dyn-clone = "1.0"
-reddsa = "0.5"
+reddsa = {version = "0.5", git = "https://github.com/heliaxdev/reddsa.git", branch = "taiga"}
```

Comment:
I'm pretty sure if you do `{ git = ..., branch = }`, then `version=...` is ignored, as it just pulls the HEAD of the branch as the dependency.

Reply:
So we'd better remove the `version =`? It seems useless if we use the branch directly?

Reply:
Yes, I think so. Should be just
`reddsa = { git = "https://github.com/heliaxdev/reddsa.git", branch = "taiga" }`

Reply:
I'll open some more topics then, as some of the other dependencies have two specifications of what the revision should be